x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/async/AsyncTaskIndexService.java

```diff
@@ -202,10 +208,12 @@ public void updateResponse(String docId,
     public void updateExpirationTime(String docId,
```

Comment:
Can you update the javadocs and maybe change the name of the function (extendExpirationTimeIfNeeded) ?

Reply:
I adjusted this in https://github.com/elastic/elasticsearch/pull/67877/commits/369cdc1748079ea0cd33c7301212d8ae5009d614